README.md

```diff
@@ -141,6 +141,87 @@ SearchResponse response = constructor.naturalLanguageSearch(request, userInfo);
+
+```java
+// Create a BrowseRequest with the filter name and filter value to request results for
+BrowseRequest request = new BrowseRequest("group_id", "8193");
+
+// Add in additional parameters
+request.setResultsPerPage(5);
+request.setPage(1);
+request.setGroupId("625");r
+request.setSortBy("Price");
+request.setSortAscending(true);
+request.getFacets().put("Brand", Arrays.asList("Jif"))
+
+// Create a UserInfo object with the session and unique device identifier (optional)
+UserInfo userInfo = new UserInfo(5, "device-id-1123123");
+userInfo.setUserSegments(Arrays.asList("Desktop", "Chrome"));
+
+// Request results as an object
+BrowseResponse response = constructor.browse(request, userInfo);
+
+// Request results as a JSON string
+String response = constructor.browseAsJSON(request, userInfo);
+```
+
+If you'd like to retrieve browse results asynchronously, the above code can be modified slightly to utilize a callback methodology:
+
+```java
+constructor.browse(request, userInfo, new BrowseCallback() {
+  @Override
+  public void onFailure(final ConstructorException exception) {
+    // failure condition
+  }
+
+  @Override
+  public void onResponse(final SearchResponse response) {
+    // success condition - data located within `response`
+  };
+});
+```
+
+# Retrieving Recommendation Results
+
+To retrieve browse results, you will need to create a `BrowseRequest`. In this request, you can also specify the number of results you want and the items (given the ids) that you want to retrieve recommendations for. If the results are for a specific user, you can also create a `UserInfo` object, which will allow you to retrieve personalized results.
+
+```java
+// Create a RecommendationsRequest with the pod id to request results for
+RecommendationsRequest request = new RecommendationsRequest("pdp_complementary_items");
+
+// Add in additional parameters
+request.setNumResults(5);
+request.setItemIds(Arrays.asList("9838172"))
+
+// Create a UserInfo object with the session and unique device identifier (optional)
+UserInfo userInfo = new UserInfo(5, "device-id-1123123");
+userInfo.setUserSegments(Arrays.asList("Desktop", "Chrome"));
+
+// Request results as an object
+RecommendationsResponse response = constructor.recommendations(request, userInfo);
+
+// Request results as a JSON string
+String response = constructor.recommendationsAsJSON(request, userInfo);
+```
+
+If you'd like to retrieve recommendation results asynchronously, the above code can be modified slightly to utilize a callback methodology:
+
+```java
+constructor.recommendations(request, userInfo, new BrowseCallback() {
```

Comment:
Still sez `Browsecallback`

Reply:
I ended up removing this portion of the README because we don't have a callback method for recommendations. I think it's something we want though right? If so, we'll add it back in with the callback changes in a separate PR if so.